README.md

```diff
@@ -12,6 +12,10 @@ folder. All TEE related code can be found in `tee`.
 
 ## Getting started
 
+If you wish to enable TLS support for the websocket connection, go to
+`demo/config.json`, and set the fields `KeyFile` to the TLS private key file
+path, and `CertFile` to the TLS certificate file path.
```

Comment:
```suggestion
If you wish to enable TLS support for the websocket connection, set the fields `KeyFile` to the TLS private key file
path, and `CertFile` to the TLS certificate file path in your configuration json file.
```
This is a general documentation about the configuration file, not specific to file `demo/config.json`. Maybe also put it in a separate sub section further down `### Configuration file`. Also mention that the configuration file can be specified for the op with cmdline flag `-config config.json`.